xml/System.Data/IDataReader.xml

```diff
@@ -306,7 +306,7 @@ Returns <see langword="null" /> if the executed command returned no resultset, o
       <Docs>
         <summary>Advances the data reader to the next result, when reading the results of batch SQL statements.</summary>
         <returns>
-          <see langword="true" /> if there are more rows; otherwise, <see langword="false" />.</returns>
+          <see langword="true" /> if there are more resultsets; otherwise, <see langword="false" />.</returns>
```

Comment:
This has already been fixed on [DbDataReader.NextResult](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbdatareader.nextresult?view=net-6.0#system-data-common-dbdatareader-nextresult) (and its async counterpart, where the term "result set" (with a space is used), could you please align with that?

Also, the summary uses the term "result"; for consistency, we should change that to:

```
Advances the data reader to the next result set, when reading the results of batch SQL statements.
```

Could you please do this here as well as on both methods on DbDataReader?

Reply:
@skorpionet100 could you please do this for DbDataReader as well, not just IDbDataReader?

Reply:
@roji done